client/package.json

```diff
@@ -4,13 +4,19 @@
   "private": true,
   "dependencies": {
     "firebase": "^5.5.3",
+    "formik": "^1.3.1",
     "node-sass": "^4.9.3",
     "react": "^16.5.2",
+    "react-date-picker": "^7.0.0",
+    "react-dnd": "^5.0.0",
+    "react-dnd-html5-backend": "^5.0.1",
     "react-dom": "^16.5.2",
     "react-icons": "^3.2.0",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.0.3",
-    "yarn-run-all": "^3.1.1"
+    "reset-css": "^4.0.1",
+    "yarn-run-all": "^3.1.1",
+    "yup": "^0.26.6"
   },
   "devDependencies": {
```

Comment:
since you're editing this file, can you update the scripts to:
```
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
```

Reply:
Have they added sass watch to the default CRA scripts?

Reply:
yeah, it "just works now" ;-)
https://facebook.github.io/create-react-app/docs/adding-a-sass-stylesheet